docs/site/src/pages/style/color/color.md

```diff
@@ -0,0 +1,27 @@
+# Color
+
+[Color](https://material.io/guidelines/style/color.html) in material design is inspired by bold hues juxtaposed with muted environments, deep shadows, and bright highlights.
+
+## Color palette
+
+This color palette comprises primary and accent colors that can be used for illustration or to develop your brand colors.
+They’ve been designed to work harmoniously with each other.
+
+For instance, you can use the red color like so:
+```js
+import { red } from 'material-ui/styles/colors';
+
+const primary = red[500]; // #F44336
+```
+
+## Color schemes
+
+We've created javascript variables for every color used in the UI Color Palette. They are stored in `styles/colors.js.`
```

Comment:
I think that this paragraph is repetitive with the previous one. I would simply remove it.